packages/build/src/lib/config/create-config.ts

```diff
@@ -4,15 +4,17 @@ import {
   DevServer,
   SwcJsMinimizerRspackPlugin,
 } from '@rspack/core';
+import { merge as rspackMerge } from 'webpack-merge';
 import { join, resolve } from 'path';
 
 export function createConfig(
   options: NgRspackPluginOptions,
-  isServer = false
+  isServer = false,
+  rspackConfigOverrides?: Partial<Configuration>
```

Comment:
Suggestion:
We could move `isServer` and `rspackConfigOverrides` into a object to have easier to maintain arguments.

This helps if one of them has default values or can be undefined.